components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/login.jsp

```diff
@@ -140,7 +138,7 @@
 
         <% if ("true".equals(loginFailed)) { %>
         <div class="alert alert-error">
-            <%=errorMessage%>
+            <%=Encode.forHtmlContent(errorMessage)%>
```

Comment:
Use Encode.forHtml here since we can embed HTML tags within &lt;div&gt; tags. So it's important to encode for all significant HTML characters